@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFCRDraw.sqf

```diff
@@ -151,30 +151,33 @@ private _pointsArray = [];
     if ((_speed >= FCR_LIMIT_MOVING_MIN_SPEED_KMH) && (_distance_m >= FCR_LIMIT_MIN_RANGE && _distance_m <= FCR_LIMIT_MOVING_RANGE)) then {
         _unitStatus = "MOVE";
     } else {
-        If (_unitType == "FLYER") then {
-            _unitType = "UNK";
+        if (_distance_m >= FCR_LIMIT_MIN_RANGE && _distance_m <= FCR_LIMIT_LOAL_LOBL_SWITCH_RANGE) then {
+            _unitStatus = "LOBL";
         };
-        if (_distance_m >= FCR_LIMIT_LOAL_LOBL_SWITCH_RANGE && _distance_m <= FCR_LIMIT_STATIONARY_RANGE) then {
+        if (_distance_m > FCR_LIMIT_LOAL_LOBL_SWITCH_RANGE && _distance_m <= FCR_LIMIT_STATIONARY_RANGE) then {
             _unitStatus = "LOAL";
-        } else {
-            _unitStatus = "LOBL";
-        }
+        };
+        if (_distance_m > FCR_LIMIT_STATIONARY_RANGE || _unitType == "FLYER") then {
+            continue;
+        };
     };
     //Unit select status
-    if (_forEachIndex == _ntsIndex) then {
-        if (_wasState == WAS_WEAPON_NONE) then {
-            _unitSelAndWpnStatus = ["NTS", "NOMSL"];
-        } else {
-            _unitSelAndWpnStatus = ["NTS"];
+    if(_unitStatus !=  "BLANK") then {
```

Comment:
Could `_unitStatus` ever be blank now? I think this if statement is now redundant